robot/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM openmicroscopy/omero-ssh-systemd
+
+MAINTAINER OME
+
+ENV JENKINS_SWARM_VERSION 2.0
+ENV LANG en_US.UTF-8
+ARG JAVAVER=openjdk18
+
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
+
+RUN yum install -y initscripts \
+    && yum clean all
+
+# Download and run omero-install.
+ENV OMERO_INSTALL /tmp/omero-install/linux
+
+RUN yum install -y git \
+    && git clone -b java-parameters https://github.com/jburel/omero-install.git /tmp/omero-install \
```

Comment:
As discussed in https://github.com/ome/omero-install/pull/94#issuecomment-206293878, using the `master` branch of `omero-install` (after merging the PR should be preferable).